### ISSUE TITLE
I've lost my admin privileges 

### ISSUE THREAD

**Issue by Brother-Andre** [Librarika/Issues#413]
I was the one who started this library: https://sbc-ihm.librarika.com/ . 

Recently, I was unable to add new users to the system, and then I realized that I was no longer a Library Admin, but only have the permissions of Library Staff, Library Assistant.

This is making it impossible to use the library. There are no other admins on our site.

**Comment by rayhan** [comment]
I believe we have closed this issue already. Please let me know if there is still any issue.

**Comment by Brother-Andre** [comment]
Dear Rayhan,

No, the issue is not at all solved. I am still only "Library Staff, Library Assistant." I must be restored to being a "Library Admin." We currently have no Library Admin, so certain things that require those permissions are impossible. 

Would you please restore me to Library Admin?

**Comment by rayhan** [comment]
Permission restored. Please check now.

**Comment by Brother-Andre** [comment]
Thank you so much, Rayhan! It works now. By the way, I figured out why this happened (I think). When I attempted to add another user, I used my own email address, because this person actually does not have her own address. Apparently, your system validates users by email address, therefore it changed my permissions to the permissions I had given the new user. 

That's just for your information. I thought it may help in the future. 

I have upgraded to the Basic Pro plan.
